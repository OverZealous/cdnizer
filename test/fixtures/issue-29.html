<script src="/node_modules/jquery/dist/jquery.js"></script>
<script src="/node_modules/backbone/backbone.js"></script>
<script src="/node_modules/lodash/lodash.js"></script>
<script src="/node_modules/handlebars/dist/handlebars.runtime.js"></script>
