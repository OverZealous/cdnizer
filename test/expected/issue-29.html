<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/15.0.0/jquery.min.js"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/backbone/15.0.0/backbone.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lodash@15.0.0/lodash.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/handlebars@15.0.0/dist/handlebars.runtime.min.js"></script>
